File: python-scripts/mysql_example.py
import mysql.connector

def test_mysql():
    try:
        connection = mysql.connector.connect(
            host='chimera-mariadb',
            user='myuser',
            password='mypassword',
            database='mydatabase'
        )
        print("Conexión exitosa a MySQL desde Python!")
        cursor = connection.cursor()
        cursor.execute("SELECT NOW()")
        row = cursor.fetchone()
        print("Hora actual en MySQL:", row[0])
    except Exception as e:
        print("Error conectando a MySQL:", e)
    finally:
        if 'connection' in locals() and connection.is_connected():
            connection.close()

if __name__ == "__main__":
    test_mysql()

File: www/mysql-example.php
<?php
$host = 'chimera-mariadb'; // nombre de servicio en docker-compose
$db   = 'mydatabase';
$user = 'myuser';
$pass = 'mypassword';

try {
    $conn = new PDO("mysql:host=$host;dbname=$db", $user, $pass);
    $conn->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);

    echo "Conexión exitosa a MySQL!<br>";
    $stmt = $conn->query("SELECT NOW() as ahora");
    $row = $stmt->fetch(PDO::FETCH_ASSOC);
    echo "Hora actual en MySQL: " . $row['ahora'];
} catch (PDOException $e) {
    echo "Error de conexión: " . $e->getMessage();
}
